firebase-firestore/src/commonTest/kotlin/dev/gitlive/firebase/firestore/GeoPointTests.kt

```diff
@@ -0,0 +1,36 @@
+package dev.gitlive.firebase.firestore
+
+import dev.gitlive.firebase.decode
+import dev.gitlive.firebase.encode
+import kotlinx.serialization.Serializable
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@Serializable
+data class TestDataWithGeoPoint(
+    val uid: String,
+    @Serializable(with = FirebaseGeoPointSerializer::class)
```

Comment:
Can you use this directly in `GeoPoint` ?

Reply:
I tried `@Contextual`, but I got an error: `KClass for Objective-C classes is not supported yet`

Reply:
Oh okay